bokehjs/src/lib/models/transforms/jitter.ts

```diff
@@ -31,10 +32,6 @@ export class Jitter extends RangeTransform {
       width:        [ p.Number, 1        ],
       distribution: [ p.Distribution, 'uniform'],
     })
-
-    this.internal({
-      previous_values: [ p.Array ],
```

Comment:
This was originally failing because `p.Array` was to strict, and I changed it before fixing that. But there also seems to be no reason for this to be a Bokeh property at all.